docs/manual/java/guide/cluster/PersistentEntityRDBMS.md

```diff
@@ -48,7 +48,7 @@ db.default {
   url = "jdbc:postgresql://database.example.com/lagom-db"
 }
 
-jdbc-defaults.slick.profile = "slick.jdbc.PostgresProfile$"
+lagom.persistence.read-side.jdbc.slick.profile = "slick.jdbc.PostgresProfile$"
```

Comment:
This doesn't look right to me. This configuration would only configure the profile for the read-side processors, not for the persistent entity.

The defaults are configured so that the original property will work, and `jdbc-defaults.slick.profile` will be used for both write-side and read-side unless you explicitly override one or the other:

https://github.com/lagom/lagom/blob/6dc90faf88ca25f4195aca75d3f22005a5cdfae9/persistence-jdbc/core/src/main/resources/reference.conf#L134-L135

Is there a specific problem that you had?